libs/blocks/notification/notification.js

```diff
@@ -90,8 +90,15 @@ function decorateFlexible(el) {
     el.querySelector('.background'),
     el.querySelector('.foreground'),
     el.querySelector('.close'),
-  ];
+  ].reduce((acc, curr) => {
+    if (curr) return [...acc, curr];
+    return acc;
+  }, []);
   const inner = createTag('div', { class: 'flexible-inner' }, innards);
+  if (el.style.background) {
+    inner.style.background = el.style.background;
+    el.style.background = '';
```

Comment:
Nit: is the empty value of use anywhere moving forward? Otherwise we could just `delete` it.

Reply:
The `CSSStyleDeclaration` interface doesn't recognize the `delete` keyword, but it does have a `removeProperty` method that achieves the same result -- I'll use that 👍 

https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty